loom/script/native/core/system/lmGC.cpp

```diff
@@ -159,7 +159,7 @@ class GC
         // it can usually have more leeway as the rate will adjust
         // under normal conditions and only sudden changes benefit from
         // a faster response.
-        bool cycleKBWarn = lastValidBPR > 0 && cycleKBDelta > cycleKB * (hibernating ? targetGarbage : cycleMemoryGrowthWarningRatio);
+        bool cycleKBWarn = cycleKB > 0 && cycleKBDelta > cycleKB * (hibernating ? targetGarbage : cycleMemoryGrowthWarningRatio);
```

Comment:
It's just three conditions and a multiplication, but fine, I can make it clearer :P